workers/worker.js

```diff
@@ -0,0 +1,18 @@
+//Create a setInterval which will keep hitting up the server every 5 minutes.
+var axios = require('axios');
+let server = '/api/worker';
+
+var worker = function(server){
+   setInterval(()=>{
+        axios.get(server)
+        .then(response =>{
+            console.log("Success!");
+        })
+        .catch(error =>{
+            console.log("ERROR!", server);
+        });
+    }, 270000);
+};
+
+
+module.exports.worker = worker;
```

Comment:
You can use just `module.exports` and then when you reference it via import you don't need to do `worker.worker.(...)` which is a little confusing.